rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -456,15 +498,47 @@ s32 cellGemGetCameraState(vm::ptr<CellGemCameraState> camera_state)
 	return CELL_OK;
 }
 
-s32 cellGemGetEnvironmentLightingColor()
+s32 cellGemGetEnvironmentLightingColor(vm::ptr<f32> r, vm::ptr<f32> g, vm::ptr<f32> b)
 {
-	UNIMPLEMENTED_FUNC(cellGem);
+	cellGem.todo("cellGemGetEnvironmentLightingColor(r=*0x%x, g=*0x%x, b=*0x%x)", r, g, b);
+	const auto gem = fxm::get<gem_t>();
+
+	if (!gem)
+	{
+		return CELL_GEM_ERROR_UNINITIALIZED;
+	}
+
+	if (!r || !g || !b)
+	{
+		return CELL_GEM_ERROR_INVALID_PARAMETER;
+	}
+
+	//if (0)
+	//{
+	//	*r = 128;
```

Comment:
May as well use 128 for the color channels (games might special-case (0,0,0)). I have it the same in my branch.

Reply:
okay I'll remove the scope and uncomment it